Libraries/src/Amazon.Lambda.Logging.AspNetCore/Amazon.Lambda.Logging.AspNetCore.csproj

```diff
@@ -4,7 +4,7 @@
 
   <PropertyGroup>
     <Description>Amazon Lambda .NET Core support - Logging ASP.NET Core package.</Description>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFrameworks>net6.0;net8.0</TargetFrameworks>
```

Comment:
Why do we need to add support for net6? Since we are doing a major version, might as well go all the way to net8

Reply:
I'm not sure what our usage is still on .NET 6. Also this change would spiral with having to remove .NET 6 from the Amazon.Lambda.AspNetCoreServer and Amazon.Lambda.AspNetCoreServer.Hosting.